pkg/controller/cluster/upgrades_test.go

```diff
@@ -0,0 +1,64 @@
+package cluster
+
+import "testing"
+
```

Comment:
Add the missing license header here @Rushmead - thanks

Reply:
Added 👍